index.js

```diff
@@ -15,63 +15,35 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  *
- * Copyright (c) 2017 Open Assessment Technologies SA;
+ * Copyright (c) 2017-2019 Open Assessment Technologies SA;
  */
 
 /**
  * CLI script entry point
  *
- * Long but linear process.
- *
  * @author Bertrand Chevrier <bertrand@taotesting.com>
  */
 
 const updateNotifier = require('update-notifier');
 
-const log = require('./src/log.js');
 const pkg = require('./package.json');
 
 updateNotifier({pkg}).notify();
 
-const argv = require('minimist')(process.argv.slice(2));
-
-const baseBranch = argv['base-branch'] || 'develop';
-const branchPrefix = argv['branch-prefix'] || 'release';
-const origin = argv['origin'] || 'origin';
-const releaseBranch = argv['release-branch'] || 'master';
-const wwwUser = argv['www-user'] || 'www-data';
-
-const release = require('./src/release')(baseBranch, branchPrefix, origin, releaseBranch, wwwUser);
-
-async function releaseExtension() {
-    try {
-        log.title('TAO Extension Release');
-
-        await release.loadConfig();
-        await release.selectTaoInstance();
-        await release.selectExtension();
-        await release.verifyLocalChanges();
-        await release.signTags();
-        await release.verifyBranches();
-        await release.initialiseGithubClient();
-        await release.doesReleaseExists();
-        await release.isReleaseRequired();
-        await release.confirmRelease();
-        await release.createReleasingBranch();
-        await release.compileAssets();
-        await release.updateTranslations();
-        await release.createPullRequest();
-        await release.extractReleaseNotes();
-        await release.mergePullRequest();
-        await release.createReleaseTag();
-        await release.createGithubRelease();
-        await release.mergeBack();
-        await release.removeReleasingBranch();
-
-        log.done('Good job!');
-    } catch (error) {
-        log.error(error);
-    }
-}
-
-releaseExtension();
+const commander = require('commander');
```

Comment:
It was the choice recommended by the spike story, and I found it very easy to work with for this project. It looks like `yargs` does many of the same things, but I don't think there is a great overhead in maintaining a basic knowledge of both command processors. But if we were going to develop many more tools, it could be something to align on.

Reply:
I don't have string opinion, but we need consistency in the tool and in the way arguments and parameters are handled

Reply:
When I created `tao-qunit-testrunner`, I used `yargs`. But it can be modified easily, if the final conclusion won't be that.

Reply:
let's see if we scale the number of node CLI tools